contents/blog/posthog-vs-optimizely.md

```diff
@@ -0,0 +1,616 @@
+---
+title: "In-depth: PostHog vs Optimizely"
+date: 2023-10-31
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+PostHog and Optimizely are both multi-product tools to help you improve your apps and websites. Two important products for each of them are experimentation and feature flags. Beyond these two, they diverge significantly:
+
+- PostHog focuses on helping you build a better product. This means having product and data tools like [product analytics](/product-analytics), [session replay](/session-replay), and [surveys](/surveys).
+- Optimizely focuses on being an all-in-one system for marketing. This means having marketing and ecommerce tools like content management, campaign planning, asset management, and checkout customizations.
+
+This post compares their platforms, experimentation features, reporting, pricing, and more.
```

Comment:
Worth doing a second pass to tighten this up a tiny bit. Just boil down the differences to the simplest, most basic concepts – i.e. PostHog is comprehensive product platform, Optimizely is marketing CTR tool, or something along along those lines.